### PR TITLE
hc-485 email validation

### PR DESCRIPTION
**Pull request by ibihim**

### CODE REVIEW

File: src/lib/validationUtils.js [hunk 1]
```diff
@@ -1,5 +1,7 @@
 /* eslint-disable no-useless-escape */
-const emailRegex = /^[a-zA-Z0-9_\-.]+@[a-zA-Z0-9\-]+\.[a-zA-Z0-9\-.]+$/;
+// https://github.com/asaskevich/govalidator/blob/v8/patterns.go#L7
+// eslint-disable-next-line max-len
```
Comment (Harsimran1) [inline src/lib/validationUtils.js]: will it take ``Dänaßäna@gmail.com`` ?

Comment (ibihim) [reply src/lib/validationUtils.js]: Yes sure, it will. I will add an appropriate test case

Comment (ibihim) [reply src/lib/validationUtils.js]: I think this changed  from not allowed to allowed:
https://en.wikipedia.org/wiki/Email_address#Internationalization

Comment (ibihim) [reply src/lib/validationUtils.js]:
```bash
 ~/workspace/tmp  node                                                                                                                      ✔  1228  09:56:05
> const emailRegex = /^((([a-zA-Z]|\d|[!#$%&'*+\-\/=?^_`{|}~]|[\u{00A0}-\u{D7FF}\u{F900}-\u{FDCF}\u{FDF0}-\u{FFEF}])+(\.([a-zA-Z]|\d|[!#$%&'*+\-\/=?^_`{|}~]|[\u{00A0}-\u{D7FF}\u{F900}-\u{FDCF}\u{FDF0}-\u{FFEF}])+)*)|((\u{22})((([\u{20}\u{09}]*(\u{0d}\u{0a}))?[\u{20}\u{09}]+)?(([\u{01}-\u{08}\u{0b}\u{0c}\u{0e}-\u{1f}\u{7f}]|\u{21}|[\u{23}-\u{5b}]|[\u{5d}-\u{7e}]|[\u{00A0}-\u{D7FF}\u{F900}-\u{FDCF}\u{FDF0}-\u{FFEF}])|(\([\u{01}-\u{09}\u{0b}\u{0c}\u{0d}-\u{7f}]|[\u{00A0}-\u{D7FF}\u{F900}-\u{FDCF}\u{FDF0}-\u{FFEF}]))))*(([\u{20}\u{09}]*(\u{0d}\u{0a}))?[\u{20}\u{09}]+)?(\u{22}))@((([a-zA-Z]|\d|[\u{00A0}-\u{D7FF}\u{F900}-\u{FDCF}\u{FDF0}-\u{FFEF}])|(([a-zA-Z]|\d|[\u{00A0}-\u{D7FF}\u{F900}-\u{FDCF}\u{FDF0}-\u{FFEF}])([a-zA-Z]|\d|-|\.|_|~|[\u{00A0}-\u{D7FF}\u{F900}-\u{FDCF}\u{FDF0}-\u{FFEF}])*([a-zA-Z]|\d|[\u{00A0}-\u{D7FF}\u{F900}-\u{FDCF}\u{FDF0}-\u{FFEF}])))\.)+(([a-zA-Z]|[\u{00A0}-\u{D7FF}\u{F900}-\u{FDCF}\u{FDF0}-\u{FFEF}])|(([a-zA-Z]|[\u{00A0}-\u{D7FF}\u{F900}-\u{FDCF}\u{FDF0}-\u{FFEF}])([a-zA-Z]|\d|-|_|~|[\u{00A0}-\u{D7FF}\u{F900}-\u{FDCF}\u{FDF0}-\u{FFEF}])*([a-zA-Z]|[\u{00A0}-\u{D7FF}\u{F900}-\u{FDCF}\u{FDF0}-\u{FFEF}])))\.?$/ui;
undefined
> emailRegex.test('Dänaßäna@gmail.com')
true
```

Comment (Harsimran1) [reply src/lib/validationUtils.js]: I like it

**Pull request merged**